### PR TITLE
Merge main into fluent2-tokens

### PR DESCRIPTION
**Pull request by laminesm**
### Platforms Impacted
- [x] iOS
- [x] macOS

### Description of changes

PR merging `main` into `fluent2-tokens`.

### Verification

(how the change was tested, including both manual and automated tests)

|Components |Main branch | Fluent2-tokens before                                       | Fluent2-tokens After                                      |
|----|----|----------------------------------------------|--------------------------------------------|
|BadgeField| main_badgefield | before_badgefield | after_badgefield |
|ObjCDemo| main_objc_demo | before_objc_demo | after_objc_demo |
|PeoplePicker| main_people_picker | before_people_picker | after_people_picker |
|ShimmerLinesView| main_shimmer | before_shimmer | after_shimmer |
|PopupMenu| main_popup_menu | before_popup_menu | after_popup_menu |

### Pull request checklist

This PR has considered:
- [x] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1358)

### CODE REVIEW

File: ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift [hunk 1]
```diff
@@ -66,9 +66,7 @@ class ShimmerViewDemoController: DemoController {
 
         container.addArrangedSubview(shimmerViewLabel("A ShimmerLinesView needs no containerview or subviews"))
         container.addArrangedSubview(dividers[0])
-        container.addArrangedSubview(ShimmerLinesView(lineCount: 3,
-                                                         firstLineFillPercent: 0.94,
-                                                         lastLineFillPercent: 0.6))
+        container.addArrangedSubview(ShimmerLinesView())
```
Comment (harrieshin) [inline ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift:69]: why we changing this?

**Review by harrieshin — APPROVED** [review]

**Comment by harrieshin** [comment]
make sure you test mac

File: ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift (hunk 1 above)
Comment (laminesm) [reply ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift:69]: I thought it'd be best to keep the shimmer demo consistent both in `main` and `fluent2-tokens`. Should I revert it?

Comment (harrieshin) [reply ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift:69]: so who is right here? main or fluent2-tokens? what cause this divergence?

Comment (laminesm) [reply ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift:69]: ShimmerLinesView without the parameters is the version from main. I don't know why there's a divergence here.

**Pull request merged**